python/tvm/relay/backend/graph_runtime_factory.py

```diff
@@ -56,6 +67,85 @@ def __init__(self, graph_json_str, libmod, libmod_name, params):
     def export_library(self, file_name, fcompile=None, addons=None, **kwargs):
         return self.module.export_library(file_name, fcompile, addons, **kwargs)
 
+    def _build_memory_map(self):
+        graph = json.loads(self.graph_json)
+
+        seen_storage_ids = set()
+        memory_map = []
+        for node_id, storage_id in enumerate(graph["attrs"]["storage_id"][1]):
```

Comment:
See my comment below -- this could be simplified if we can have relay.Expr --> sid Map somewhere accessible and use that to create the json later while this function also being another consumer of that map rather than parsing the json and extracting size information out of it. WDYT?

Reply:
I agree with that, see other comment